assets/js/modules/analytics/datastore/properties.test.js

```diff
@@ -110,15 +103,14 @@ describe( 'modules/analytics properties', () => {
 					message: 'Internal server error',
 					data: { status: 500 },
 				};
-
-				fetch
-					.doMockIf(
-						/^\/google-site-kit\/v1\/modules\/analytics\/data\/create-property/
-					)
-					.mockResponse(
-						JSON.stringify( response ),
-						{ status: 500 }
-					);
+				fetchMock.mock(
+					/^\/google-site-kit\/v1\/modules\/analytics\/data\/create-property/,
+					{ body: response, status: 500 }
+				);
+				fetchMock.once(
+					/^\/google-site-kit\/v1\/modules\/analytics\/data\/properties-profiles/,
+					{ body: {}, status: 200 }
+				);
```

Comment:
This mock wasn't there before and shouldn't be necessary. Are you finding an unmatched request warning without this?

Reply:
@aaemnnosttv  yeah, there were quite a few tests in which there were unmatched request warnings and console.errors (mostly non-failing) so if there are any extra mocks they're to prevent those. Maybe these warrant inline comments just to explain why they're there?